data/starts.txt

```diff
@@ -0,0 +1,227 @@
+	conversation "hai intro"
+	account
+		credits 1431000
+		score 400
+		mortgage Mortgage
+			principal 1431000
+			interest 0.003
+			term 730
+	set "hai start"
+	# need to set it here to not have it trigger alongside the invisible mission
+	set "First Contact: Hai: offered"
+	set "Intro [0]: offered"
+	set "skip FW"
+	set "skip HR"
+
+
+
+conversation "hai intro"
+	scene "scene/hai shipyard"
+	`	You've never been to the shipyard before, and as you arrive, it already feels like you're traveling to another world. The pictures and video you've seen of all these ships before simply don't do them justice; seeing them for real is something else entirely. Even though you know a lot of those ships must have been in service for many years, you are stunned by how new and shiny they look. You begin to wonder if you are ready to take the big leap.`
+	`	This "Aphid" could suit your needs. It is fast and able to carry a respectable amount of cargo.`
+	scene "thumbnail/hai aphid"
+	`	That is, unless you want to get adventurous and try out the impetuous Grasshopper. Surely the interceptor would be able to beat the smaller human warships, provided a competent pilot is at the helm.`
+	`	You have, after all, heard tales of pirates and quick money... And up north you can find the 'Unfettered'. Maybe they could come to like you, if you were to show them the strength and cunning they respect.`
+	choice
+		`	(Go to the bank.)`
+	scene "scene/hai bank"
+	`	You head to the bank's lobby, hoping to get enough of a loan to be able to buy one of these pristine ships.`
+	`	To your disapointment the only ship you could afford is an Aphid. You arrange the contract to loan as much money as you need in order to buy one, in what has to be the first seriously unwise act of your life. You may have to sell some part of the ship in order to be able to afford flying it!`
+	`	You can probably sell the shield regenerator without any consequences, so long as you do not get too close to hostile territory. You may even downgrade the anti-missile turret.`
+	`	The bank employee sums it up to you, with a hesitant smile. "This will put you in a debt of 1 million 431 thousand credits, due in the course of 2 years, with an interest rate of 0.3% per day. Please sign here."`
+	name
+	`	You head out of the bank, but on your way to buy the Aphid at the shipyard, you see a notice. A retiring captain is offering to sell some of his ships at an interesting price, including a Grasshopper... Maybe they could sell it to you.`
+	choice
+		`	(Go to the shipyard.)`
+			decline
+		`	(Go see what the captain has to offer first.)`
+	`	You head to the spaceport and easily find the captain selling off his ships. Thankfully they did not yet sell the Grasshopper.`
+	`	The captain greets you in Hai, but you've somehow managed to never really learn the language. Most everyone speaks yours to at least some degree anyway, and once they realize you do not understand, they switch with little effort.`
```

Comment:
```suggestion
	`	The captain greets you in Hai, but you've somehow managed to never really learn the language. Most people speaks yours to at least some degree anyway, and once they realize you do not understand them, they switch with little effort.`
```
This paragraph needs some more work.

Reply:
`	The captain greets you in Hai, but when you explain to him in your language that you never learned Hai, he switches to speaking relatively fluidly in your language.`

Reply:
```
`	The captain greets you in Hai. You are embarrassed to have to explain to them that you never learned their language. Most Hai speak your language however, and, as usual, they are able to switch to speaking yours without much difficulty.`
```
Still not sure.